bindings/python/benchmarks/benchmarks.py

```diff
@@ -197,6 +220,7 @@ def exercise_table(table):
         ]
 
     # All of the following tests are being skipped because NumPy/Pandas do not work with nested arrays.
+    # todo - Polars DOES have a Struct type. Try implementing tests for it.
```

Comment:
There are some TODO comments in this file, can you remove them and file a ticket instead?

Reply:
Removed  [ARROW-217](https://jira.mongodb.org/browse/ARROW-217)